sources/packages/api/src/esdc-integration/e-cert-integration/models/e-cert-integration-model.ts

```diff
@@ -38,7 +38,7 @@ export interface ECertRecord {
   addressLine1: string;
   addressLine2: string;
   city: string;
-  province: string;
```

Comment:
Were we passing the postal code data to province key before?

Reply:
good catch, its `provinceState`